statsmodels/duration/hazard_regression.py

```diff
@@ -324,12 +326,16 @@ def __init__(self, endog, exog, status=None, entry=None,
             self.offset = np.asarray(self.offset)
 
         self.surv = PHSurvivalTime(self.endog, self.status,
-                                    self.exog, self.strata,
-                                    self.entry, self.offset)
+                                   self.exog, self.strata,
+                                   self.entry, self.offset)
 
         # TODO: not used?
         self.missing = missing
 
+        self.df_resid = (np.float(self.exog.shape[0] -
+                                  np_matrix_rank(self.exog)))
+        self.df_model = np.float(np_matrix_rank(self.exog) - 1)
```

Comment:
This was a mistake.  There's no intercept in a Cox model, so should not subtract 1.